### PR TITLE
Use Puppeteer instead of pdf-puppeteer.  Document usage in Docker

### PR DESCRIPTION
**Pull request by atodd-noms**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,13 @@ app.use('/pdf', (req, res) => {
 })
 ```
 
+### Configuring Puppeteer
+Pass args to the Puppeteer launch as follows. See [Puppeteer launch options](https://github.com/GoogleChrome/puppeteer/blob/v1.18.1/docs/api.md#puppeteerlaunchoptions) for more info
+```javascript
+const puppeteerArgs = { args: ['--no-sandbox', '--disable-setuid-sandbox'] }
```
Comment (romidane) [inline README.md]: probably worth defaulting this and allowing the user to install change them if they want to

**Review by romidane — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -34,6 +34,26 @@ app.use('/pdf', (req, res) => {
...
+
+Default options:
+```javascript
+{ args: ['--no-sandbox', '--disable-setuid-sandbox'] }
+``` 
+
+Pass options to the pdfRendered function to replace the defaults. Pass an empty object if you only want to remove the defaults.
+See [Puppeteer launch options](https://github.com/GoogleChrome/puppeteer/blob/v1.18.1/docs/api.md#puppeteerlaunchoptions) for more info
+```javascript
+app.use(pdfRenderer( { timeout:60000 } ))
```
Comment (romidane) [inline README.md]: Can we also default the timeout?

Comment (atodd-noms) [reply README.md]: There already is a default of 30000 (30 seconds). This was just meant to be an example of how you could set other options

Comment (atodd-noms) [reply README.md]: Clarified the readme

**Pull request merged**